docs/reference/api/connectivity/lorawan/lorawan.md

```diff
@@ -0,0 +1,70 @@
+<h2 id="lorawan-api">LoRaWAN</h2>
+
+LoRaWAN is a technology designed for low-power battery powered devices. These devices operate in an unlicensed spectrum, creating high desnity wide-area networks.
+
+Arm Mbed OS provides a native network stack for LoRaWAN, which can run on any Mbed Enabled device with a LoRa radio onboard. 
+
+The [LoRaWANInterface](https://github.com/ARMmbed/mbed-os/blob/feature-lorawan/features/lorawan/LoRaWANInterface.h) provides a C++ API for connecting to the internet over a LoRa network.
+
+## LoRaWANInterface class reference
+
+[![View code](https://www.mbed.com/embed/?type=library)](http://os-doc-builder.test.mbed.com/docs/v5.8/mbed-os-api-doxy/class_l_o_r_a_w_a_n_interfcae.html)
```

Comment:
Note: This link doesn't work even after I update the spelling to "interface.html" Please fix it or let me know when it will go into effect.

Reply:
oh. The spellings :) I wrote this document when I was travelling and had terrible flu :) Affects are pretty obvious. Yeah, the link may work only after the release. I am not actually sure how this works. My assumption is that when you will update docs for 5.8 (during release process), a script will fetch doxygen and this link would start working.